src/overlays/actors/ovl_En_Zl4/z_en_zl4.c

```diff
@@ -1,7 +1,7 @@
 /*
  * File: z_en_zl4.c
  * Overlay: ovl_En_Zl4
- * Description:
+ * Description: Glitched early version of Skull Kid stuck in a T-pose
```

Comment:
This description is identical to `z_dm_sa.c`, are they both like this?

Reply:
Yeah, they both are basically the same thing. They are not *exactly* the same thing, since one of them (I can't remember which) will render him without a flute with certain params, while the other will always render him with the flute. But either way, they're glitchy actors stuck in a T-pose.